video/getting-started/create-video-client/create-video-client.js

```diff
@@ -1,10 +1,10 @@
 // If you are requiring twilio-video from CommonJS,
```

Comment:
I think "requiring" is a little more direct than "use", and makes sense in the context of CommonJS.